README.md

```diff
@@ -73,7 +73,7 @@ you can build a docker image. Subsequently, by running:
 ```shell
 $ docker run -p 3000:80 publiccode-editor
```

Comment:
I believe the switch should be this one 😄 @bfabio Could you please commit this change? Thanks!
```suggestion
$ docker run -p 80:3000 publiccode-editor
```

Reply:
@bfabio Nevermind, I didn't check the new docker-compose.yml, sorry. I think your suggestion makes sense, I leave the review to @LucaPrete who is following the new infra.